src/components/card/Card/Card.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import classnames from 'classnames'
+
+import { GridLayoutProp, applyGridClasses } from '../../grid/Grid/Grid'
+
+interface CardProps {
+  layout: 'standardDefault' | 'flagDefault' | 'flagMediaRight'
+  headerFirst?: boolean
+}
+
+export const Card = (
+  props: CardProps & React.HTMLAttributes<HTMLDivElement> & GridLayoutProp
+): React.ReactElement => {
+  const {
+    layout,
+    headerFirst,
+    children,
+    className,
+    gridLayout,
+    ...divProps
+  } = props
+
+  // TODO determine how to implement the tablet:grid-col-4 class properly, should this be li class?
+  const gridClasses = gridLayout && applyGridClasses(gridLayout)
+
+  const classes = classnames(
+    'usa-card',
+    {
+      'usa-card--header-first': headerFirst,
+      'usa-card--flag': layout === 'flagDefault' || layout === 'flagMediaRight',
+      'usa-card--media-right': layout === 'flagMediaRight',
+    },
+    gridClasses,
+    className
+  )
+
+  return (
+    <li className={classes} data-testid="Card">
+      <div className="usa-card__container" {...divProps}>
```

Comment:
There's a slight inconsistency here that I could see being confusing. take this example:
```
<Card className="my-custom-class" id="myCard">
```
will result in the `className` on the `<li>` and the `id` on the `<div>`:
```
<li className="my-custom-class">
  <div id="myCard">
```

While I agree it may be useful to pass in attributes for the underlying `div`, maybe for this first implementation let's keep it simple and say the implicit props are all for the `li` instead?

Reply:
after adding more examples with Storybook I'm gonna say this is a higher priority change.. if it sounds okay with you @eamahanna I think this should get in before being merged